sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorDispatcher.java

```diff
@@ -257,7 +257,7 @@ public void run(Selectable selectable) {
                     ioSignal.source().close();
                 }
             } catch (IOException ioException) {
-                logger.error("CloseHandler.source().close() failed with an error.", ioException);
+                logger.warning("CloseHandler.source().close() failed with an error.", ioException);
```

Comment:
My understanding is, it’s less likely to hit this code path (so the log line) and line 250, but when it does, this error likely to be useful for us, to back track any memory-leak due to io-pipes not being cleaned up?

Reply:
TBH; I've yet to see this in any incident logs.

Reply:
Good point! 

Do you think people would disable warnings coming from our loggers? 

the main reason I change closing errors to warnings is that when things are being closed, we create a ton of logs which look scary but are probably benign. 

But based on you explanation, I realized this is an error from close() and not some race-condition because something is already closed. So I'm happy to revert changes here.

Reply:
Typically, users want to use the lowest logging level. Upon request, _warn_ is someone less hesitant to set compared to _verbose_, upon request. Yes, given we never observed this error, I was thinking keeping it as it is